front/app/containers/Admin/projects/edit/messages.ts

```diff
@@ -103,7 +103,7 @@ export default defineMessages({
   createSurveyDescription: {
     id: 'app.containers.AdminPage.ProjectEdit.createSurveyDescription',
     defaultMessage:
-      'Embed a Typeform, Google Form, Enalyzer or SurveyXact survey.',
+      'Embed a Typeform, Google Form, Enalyzer, SurveyXact or Qualtrics survey.',
```

Comment:
Complete Crowdin noob here, but don't we have to modify this message in the en translation file as well to trigger new translations.